src/index.jsx

```diff
@@ -1,10 +1,11 @@
+import React from 'react'
 import core from './core'
 import TailSpin from './TailSpin'
 
 export default (
   ComposedComponent,
   {
-    LoadingIndicator = TailSpin,
-    ...rest,
+    renderLoadingIndicator = (() => <TailSpin />),
```

Comment:
Although, once again, there is not a way to add props to the spinner separately from the ones for the component... :/